packages/graphback-codegen-schema/src/SchemaCRUDPlugin.ts

```diff
@@ -421,53 +417,70 @@ export class SchemaCRUDPlugin extends GraphbackPlugin {
-    if (model.crudOptions.findOne) {
-      this.addFindOneQueryResolver(model, queryObj)
-    }
-    if (model.crudOptions.find) {
-      this.addFindQueryResolver(model, queryObj)
+  protected addQueryResolvers(model: ModelDefinition, resolvers: IResolvers) {
+    if (model.crudOptions.findOne || model.crudOptions.find) {
+      resolvers.Query = {} as IObjectTypeResolver
+
+      if (model.crudOptions.findOne) {
+        this.addFindOneQueryResolver(model, resolvers.Query)
+      }
+      if (model.crudOptions.find) {
+        this.addFindQueryResolver(model, resolvers.Query)
+      }
     }
   }
 
   /**
    * Create Mutation resolver fields
    *
    * @param {ModelDefinition} model - The model definition with CRUD config and GraphQL typr
-   * @param {IFieldResolver} mutationObj - Mutation resolver object
+   * @param {IResolvers} resolvers - root resolver object
    */
-  protected addMutationResolvers(model: ModelDefinition, mutationObj: IObjectTypeResolver) {
-    if (model.crudOptions.create) {
-      this.addCreateMutationResolver(model, mutationObj)
-    }
-    if (model.crudOptions.update) {
-      this.addUpdateMutationResolver(model, mutationObj)
-    }
-    if (model.crudOptions.delete) {
-      this.addDeleteMutationResolver(model, mutationObj)
+  protected addMutationResolvers(model: ModelDefinition, resolvers: IResolvers) {
+    if (model.crudOptions.create || model.crudOptions.update || model.crudOptions.delete) {
+      resolvers.Mutation = {} as IObjectTypeResolver
+
+      if (model.crudOptions.create) {
+        this.addCreateMutationResolver(model, resolvers.Mutation)
+      }
+      if (model.crudOptions.update) {
+        this.addUpdateMutationResolver(model, resolvers.Mutation)
+      }
+      if (model.crudOptions.delete) {
+        this.addDeleteMutationResolver(model, resolvers.Mutation)
+      }
     }
   }
 
   /**
    * Create Subscription resolver fields
    *
    * @param {ModelDefinition} model - The model definition with CRUD config and GraphQL typr
-   * @param {IFieldResolver} subscriptionObj - Subscription resolver object
+   * @param {IResolvers} resolvers - root resolver object
    */
-  protected addSubscriptionResolvers(model: ModelDefinition, subscriptionObj: IObjectTypeResolver) {
+  protected addSubscriptionResolvers(model: ModelDefinition, resolvers: IResolvers) {
     const modelType = model.graphqlType;
 
-    if (model.crudOptions.create && model.crudOptions.subCreate) {
-      this.addCreateSubscriptionResolver(modelType, subscriptionObj)
-    }
-    if (model.crudOptions.update && model.crudOptions.subUpdate) {
-      this.addUpdateSubscriptionResolver(modelType, subscriptionObj)
-    }
-    if (model.crudOptions.delete && model.crudOptions.subDelete) {
-      this.addDeleteSubscriptionResolver(modelType, subscriptionObj)
+    const createSubEnabled = model.crudOptions.create && model.crudOptions.subCreate;
```

Comment:
I would suggest to remove create from it. This is use case we going to have - we want to have just subscriptions enabled for the object but no create.

Good timing

Reply:
Sure!